### PR TITLE
[Merged by Bors] - Cargo fmt with unstable features

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Fresh version of #1670 off the latest main.

Mostly fixing documentation wrapping.

### CODE REVIEW

**Comment by alice-i-cecile** [comment]
This is just `cargo fmt` using the unstable cargo fmt features from rustfmt.toml.

File: crates/bevy_render/src/mesh/mesh.rs [hunk 1]
```diff
@@ -237,11 +237,10 @@ impl Mesh {
     /// The direction the vertex normal is facing in.
     /// Use in conjunction with [`Mesh::set_attribute`]
     pub const ATTRIBUTE_NORMAL: &'static str = "Vertex_Normal";
-    /// The direction of the vertex tangent. Used for normal mapping
-    pub const ATTRIBUTE_TANGENT: &'static str = "Vertex_Tangent";
-
     /// Where the vertex is located in space. Use in conjunction with [`Mesh::set_attribute`]
     pub const ATTRIBUTE_POSITION: &'static str = "Vertex_Position";
+    /// The direction of the vertex tangent. Used for normal mapping
+    pub const ATTRIBUTE_TANGENT: &'static str = "Vertex_Tangent";
```
Comment (bjorn3) [inline crates/bevy_render/src/mesh/mesh.rs:243]: This should stay next to normal.

Comment (alice-i-cecile) [reply crates/bevy_render/src/mesh/mesh.rs:243]: How do I signal to cargo fmt not to immediately make this change again?

Comment (bjorn3) [reply crates/bevy_render/src/mesh/mesh.rs:243]: `#[rustfmt::ignore]` maybe?

Comment (alice-i-cecile) [reply crates/bevy_render/src/mesh/mesh.rs:243]: Nope, crashes with error: invalid attribute.

Looks like `#[rustfmt::skip]` is the correct incantation.

Comment (alice-i-cecile) [reply crates/bevy_render/src/mesh/mesh.rs:243]: Which, of course, doesn't actually stop it from reordering. @bjorn3, before I waste an hour on this, how much do you care?

Comment (DJMcNab) [reply crates/bevy_render/src/mesh/mesh.rs:243]: I personally think we could just as easily disable the option - 7212b70478c8465b178611794d5738703eb9f6f1 doesn't explain why it was enabled.

Additionally, it currently seems like the option needs work, see https://github.com/rust-lang/rustfmt/issues/2792#issuecomment-397917400

Comment (alice-i-cecile) [reply crates/bevy_render/src/mesh/mesh.rs:243]: I agree with you there. @cart, do we have your blessing?

Comment (cart) [reply crates/bevy_render/src/mesh/mesh.rs:243]: Yup lets just disable it. Using order to semantically group things does seem valid in some contexts.

Comment (alice-i-cecile) [reply crates/bevy_render/src/mesh/mesh.rs]: This option has been removed!

**Comment by alice-i-cecile** [comment]
@cart this is done now with the option removed. Please merge this ASAP so I can stop rebasing <3

**Pull request closed without merging**